FandomParser/Tests/InfoboxParser.Tests/InfoboxParser.Tests.csproj

```diff
@@ -50,6 +50,42 @@
     <None Update="Testdata\empty_BothWorlds.infobox">
       <CopyToOutputDirectory>Always</CopyToOutputDirectory>
     </None>
+    <None Update="Testdata\Extractor\input_Bakery.txt">
```

Comment:
As these are all the same, and we are using the new format, we can use 
```xml
<None Update="Testdata\**\*.txt">
     <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
</None>
<None Update="Testdata\**\*.infobox">
     <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
</None>
```
or possibly just the following, although this is not tested
```xml
<None Update="Testdata\**\*.txt, Testdata\**\*.infobox">
     <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
</None>
```

and remove all the individual entries.